.github/workflows/task-sdk-tests.yml

```diff
@@ -40,6 +40,10 @@ on:  # yamllint disable-line rule:truthy
         description: "JSON-formatted array of Python versions to build images from"
         required: true
         type: string
+      run-task-sdk-tests:
```

Comment:
@potiuk This was meant to fix the "task sdk" workflow never called :)

Reply:
Coz it already has the following in Task SDK workflow

https://github.com/apache/airflow/blob/899dcbfcb2b0ca2a8eb50cfb807bf96c336a3552/.github/workflows/task-sdk-tests.yml#L61